runtime/src/snapshot_utils.rs

```diff
@@ -855,6 +839,77 @@ pub fn bank_from_snapshot_archives(
             None
         };
 
+    Ok((unarchived_full_snapshot, unarchived_incremental_snapshot))
+}
+
+pub fn bank_fields_from_snapshot_archives(
+    // Pass in path to the directory containing the snapshot archive
+    bank_snapshots_dir: impl AsRef<Path>,
+) -> Result<BankFieldsToDeserialize> {
+    let full_snapshot_archive_info = get_highest_full_snapshot_archive_info(&bank_snapshots_dir)
```

Comment:
Since the bank snapshots and the snapshot archives can be different directories, there should be separate parameters for both. 

For here, please add a new parameter `full_snapshot_archives_dir: impl AsRef<Path>` and use that here.

Reply:
done!